src/Debug.elm

```diff
@@ -56,12 +56,18 @@ crash = Native.Debug.crash
 
 {-| Watch a particular value in the debugger. Say we want to know the value of
 a variable called `velocity` because it may not be updated correctly. Adding
-`Debug.watch` allows us to name the value and show it with the debugger.
+`Debug.watch` allows us to name the value and show it with the debugger. The
+result of evaluating such an expression is exactly the same as if not having
+the call to `Debug.watch` at all.
```

Comment:
Remove "if". Also, "the result of evaluating such an expression" is a bit technical. Maybe just "the value is returned unchanged"?